.prettyignore

```diff
@@ -0,0 +1,4 @@
+# Ignore artifacts:
+build
+coverage
+
```

Comment:
Make sure to fix the file name to `.prettierignore`

Reply:
Fixed now.